projects/plugins/crm/src/automation/class-automation-engine.php

```diff
@@ -8,32 +8,42 @@
 namespace Automattic\Jetpack\CRM\Automation;
 
 /**
- * Automation Engine
+ * Automation Engine.
  *
- * @package Automattic\Jetpack\CRM\Automation
+ * @since $$next-version$$
  */
 class Automation_Engine {
 
-	/** @var Automation_Engine Instance singleton */
+	/**
+	 * @var Automation_Engine Instance singleton.
+	 */
 	private static $instance = null;
 
-	/** @var array triggers map name => classname */
+	/**
+	 * @var array triggers map name => classname.
```

Comment:
This isn't really part of the WP CS but my own preference. If we know that an array is a list of the same type of "thing", then we might as well define it so IDE's can better understand it - and may even support auto complete/suggestion
```suggestion
	 * @var string[] triggers map name => classname.
```